website/docs/idx-terminology.md

```diff
@@ -25,11 +25,15 @@ A **Document** is a representation of structured data on the [Ceramic network](#
 
 A **Document Identifier (DocID)** is a URL representing a [Document](#document). Multiple APIs in the [IDX library](libs-idx.md) use DocIDs as inputs and/or outputs.
 
+## IndexKey
+
+An **Index Key** is a key in the [Identity Index](#identity-index--idx). It should be treated as an opaque string.
+
 ## Identity Index / IDX
 
 The **Identity Index** and **IDX** can sometimes both be used to refer to the [Identity Index specification CIP](https://github.com/ceramicnetwork/CIP/issues/3), but in this documentation the term **Identity Index** will always be used to refer to the [specification CIP](https://github.com/ceramicnetwork/CIP/issues/3), while **IDX** refers to the library, or possibly the [`IDX` class](libs-idx.md#idx-class) in code snippets.
 
-The **Identity Index contents** refers to the mapping of a [Definition](#definition), represented by its [DocID](#docid), to an [Entry](#entry).
+The **Identity Index contents** refers to the mapping of a [Definition](#definition), represented by its [IndexKey](#indexkey), to a referenced [DocID](#docid).
```

Comment:
I don't think we need to actually expose this concept to the developer. But fine for now. Let's revisit when we implement encryption.